api/resolvers/item.js

```diff
@@ -80,6 +80,8 @@ const orderByClause = (by, me, models, type) => {
       return 'ORDER BY "Item".msats DESC'
     case 'zaprank':
       return topOrderByWeightedSats(me, models)
+    case 'random':
+      return 'ORDER BY "Item".msats DESC'
```

Comment:
This ORDER BY is just a placeholder - can't get RAND() or NEWID() to work here. Get a 404 if the query doesn't work.

Reply:
`RAND()` is a MySQL thing, PostgreSQL uses `RANDOM()`, see https://www.postgresql.org/docs/13/functions-math.html

Reply:
Ah right. I'll try this.

Reply:
Yeah this works 👍